pkg/apis/pipeline/v1beta1/pipelinerun_validation.go

```diff
@@ -129,20 +129,15 @@ func validateSpecStatus(ctx context.Context, status PipelineRunSpecStatus) *apis
 	case PipelineRunSpecStatusCancelled,
 		PipelineRunSpecStatusCancelledRunFinally,
 		PipelineRunSpecStatusStoppedRunFinally:
-		return ValidateEnabledAPIFields(ctx, "graceful termination", "alpha")
+		return nil
 	}
 
-	cfg := config.FromContextOrDefaults(ctx)
-	if cfg.FeatureFlags.EnableAPIFields == config.AlphaAPIFields {
-		return apis.ErrInvalidValue(fmt.Sprintf("%s should be %s, %s, %s or %s", status,
-			PipelineRunSpecStatusCancelled,
-			PipelineRunSpecStatusCancelledRunFinally,
-			PipelineRunSpecStatusStoppedRunFinally,
-			PipelineRunSpecStatusPending), "status")
-	}
-	return apis.ErrInvalidValue(fmt.Sprintf("%s should be %s or %s", status,
```

Comment:
It seems like removing this will mean that the deprecated value will no longer work, and it seems like from previous PR comments that we want to avoid removing this value until the new value is promoted to beta.

Reply:
@lbernick it's just the webhook validation which was checking for the fields that were behind feature flag.

Reply:
@lbernick it's just the error message. We could still include `PipelineRunSpecStatusCancelledDeprecated` but as it is deprecated, I don't really think it would make sense.